kratos/processes/replace_elements_and_condition_process.cpp

```diff
@@ -109,35 +115,31 @@ void UpdateConditionsInSubModelPart(
 
 void ReplaceElementsAndConditionsProcess::Execute()
 {
-    const std::string element_name = mSettings["element_name"].GetString();
-    const std::string condition_name = mSettings["condition_name"].GetString();
-    ModelPart& root_model_part = mrModelPart.GetRootModelPart();
+    const std::string& r_element_name = mSettings["element_name"].GetString();
+    const std::string& r_condition_name = mSettings["condition_name"].GetString();
+    ModelPart& r_root_model_part = mrModelPart.GetRootModelPart();
 
-    if (element_name != "") {
+    if (r_element_name != "") {
         std::unordered_set<std::size_t> set_element_ids (mrModelPart.NumberOfElements());
-        for(auto& ielem : mrModelPart.Elements()) {
-            set_element_ids.insert(ielem.Id());
+        for(auto& r_elem : mrModelPart.Elements()) {
```

Comment:
```suggestion
        for(const auto& r_elem : mrModelPart.Elements()) {
```
I think this can be `const`.

Reply:
I will check